common/constants.js

```diff
@@ -246,12 +253,14 @@ module.exports = Object.freeze({
     RESOURCE_GROUPS: {
       LOCK: 'lock.servicefabrik.io',
       DEPLOYMENT: 'deployment.servicefabrik.io',
-      BACKUP: 'backup.servicefabrik.io'
+      BACKUP: 'backup.servicefabrik.io',
+      RESTORE: 'backup.servicefabrik.io'
```

Comment:
`restore.servicefabrik.io` ?

Reply:
No, the resource Group for restore is backup.